apps/profiles/serializers/profiles.py

```diff
@@ -81,7 +84,8 @@ def update(self, instance, validated_data):
             change_followers_if_exists(instance)
 
         categories = validated_data.pop("category_favorite", [])
-
+        if validated_data.get("city_location"):
+            utils.update_city_if_exist(instance=instance, validated_data=validated_data)
         profile: User = super().update(instance, validated_data)
```

Comment:
we need to at least manually test update since we don't have tests on update case

Reply:
Add tests for one